### PR TITLE
added list/array support

### PR DESCRIPTION
**Pull request by DvirDukhan**
added basic array support

### CODE REVIEW

File: redisgraph/query_result.py [hunk 1]
```diff
@@ -9,12 +9,15 @@ class ResultSetColumnTypes(object):
...
-    PROPERTY_INTEGER = 3
-    PROPERTY_BOOLEAN = 4
-    PROPERTY_DOUBLE = 5
+    VALUE_UNKNOWN = 0
+    VALUE_NULL = 1
+    VALUE_STRING = 2
+    VALUE_INTEGER = 3
+    VALUE_BOOLEAN = 4
+    VALUE_DOUBLE = 5
+    VALUE_ARRAY = 6
```
Comment (swilly22) [inline redisgraph/query_result.py]: Are VALUE_EDGE and VALUE_EDGE new?
if not VALUE_ARRAY must come after them

File: test.py [hunk 2]
```diff
@@ -33,6 +33,10 @@ def test_graph_creation(self):
         self.assertEqual(visit.properties, edge.properties)
         self.assertEqual(country, japan)
 
+        query = """RETURN [1, 2.3, "4", true, false, null]"""
```
Comment (swilly22) [inline test.py]: Include a sub-array within this array.

File: redisgraph/query_result.py [hunk 3]
```diff
@@ -2,19 +2,25 @@
 from .edge import Edge
 from prettytable import PrettyTable
 
+
```
Comment (swilly22) [inline redisgraph/query_result.py]: Remove extra line

File: redisgraph/query_result.py [hunk 4]
```diff
@@ -2,19 +2,25 @@
...
 class ResultSetColumnTypes(object):
     COLUMN_UNKNOWN = 0
     COLUMN_SCALAR = 1
     COLUMN_NODE = 2
     COLUMN_RELATION = 3
 
+
```
Comment (swilly22) [inline redisgraph/query_result.py]: Remove extra line

File: redisgraph/query_result.py [hunk 5]
```diff
@@ -34,11 +40,11 @@ def __init__(self, graph, response):
             self.parse_statistics(response[0])
         else:
             self.parse_results(response)
-            self.parse_statistics(response[-1]) # Last element.
+            self.parse_statistics(response[-1])  # Last element.
```
Comment (swilly22) [inline redisgraph/query_result.py]: Remove extra space

File: redisgraph/query_result.py [hunk 6]
```diff
@@ -136,10 +143,21 @@ def parse_scalar(self, cell):
             else:
                 print("Unknown boolean type\n")
 
-        elif scalar_type == ResultSetScalarTypes.PROPERTY_DOUBLE:
+        elif scalar_type == ResultSetScalarTypes.VALUE_DOUBLE:
             scalar = float(value)
 
-        elif scalar_type == ResultSetScalarTypes.PROPERTY_UNKNOWN:
+        elif scalar_type == ResultSetScalarTypes.VALUE_ARRAY:
+            scalar = value
```
Comment (swilly22) [inline redisgraph/query_result.py]: Consider renaming value to array for clarity

File: redisgraph/query_result.py (hunk 5 above)
Comment (DvirDukhan) [reply redisgraph/query_result.py]: done

File: redisgraph/query_result.py (hunk 4 above)
Comment (DvirDukhan) [reply redisgraph/query_result.py]: PEP 8 formatting

File: redisgraph/query_result.py (hunk 3 above)
Comment (DvirDukhan) [reply redisgraph/query_result.py]: PEP 8 formatting

File: redisgraph/query_result.py (hunk 6 above)
Comment (DvirDukhan) [reply redisgraph/query_result.py]: done

File: test.py [hunk 7]
```diff
@@ -33,8 +33,37 @@ def test_graph_creation(self):
...
+
+        query = """MATCH(n) return collect(n) as x"""
+        result = redis_graph.query(query)
+
+        a = Node(label='person', properties={'name': 'a', 'age': 32, 'array': [0, 1, 2]})
+        b = Node(label='person', properties={'name': 'b', 'age': 30, 'array': [3, 4, 5]})
+
+        self.assertEqual([a, b], result.result_set[0][0])
+
+
```
Comment (swilly22) [inline test.py]: Remove extra line

**Pull request merged**